mix.exs

```diff
@@ -38,7 +39,8 @@ defmodule CxLeaderboard.MixProject do
     [
       {:dialyxir, "~> 0.5", only: :dev, runtime: false},
       {:benchee, "~> 0.12", only: :dev, runtime: false},
-      {:ex_doc, "~> 0.18", only: :dev, runtime: false}
+      {:ex_doc, "~> 0.18", only: :dev, runtime: false},
+      {:redix, ">= 0.0.0"}
```

Comment:
Is there any way we can make redix not a dependency unless someone wants to use Redis store?

Reply:
I'm not as familiar with idiomatic elixir, but i can take a look!